terraform/aptos-node-testnet/main.tf

```diff
@@ -56,7 +56,7 @@ module "aptos-node-helm-values-deepmerge" {
   # https://registry.terraform.io/modules/Invicton-Labs/deepmerge/null/0.1.5
   source = "Invicton-Labs/deepmerge/null"
   maps = [
-    var.enable_forge ? local.aptos_node_helm_values_forge_override : map(),
+    jsondecode(var.enable_forge ? jsonencode(local.aptos_node_helm_values_forge_override) : jsonencode({})),
```

Comment:
don't really get what's happening in this line.. we're applying `jsondecode` upwards of 3 times -- is it idempotent? there's no way to specify an empty map ?

Reply:
Apparently not, if you run this locally you should be able to repro and get this error

```
│ The true and false result expressions must have consistent types. The 'true' value includes object attribute "fullnode", which is absent in the 'false' value.
```

So oyu could make an empty map which matches the structure or you can do this to strip the type, perhaps theres an easier way to ignore / strip type but I dont know, I just tried this

Reply:
hmm ok gotcha. can you add a small comment to the code describing the approach?